recipes-rubygems/rubygems-nokogiri_1.12.3.bb

```diff
@@ -4,12 +4,13 @@ DESCRIPTION = "Nokogiri (鋸) makes it easy and painless to work with XML and HT
 HOMEPAGE = "https://nokogiri.org"
 
 LICENSE = "MIT"
-LIC_FILES_CHKSUM = "file://LICENSE-DEPENDENCIES.md;md5=2fcfc4989eb71e592d2e85d26d7fa91c"
```

Comment:
potential issue: Variable 'HOMEPAGE' is set by rubygems-nokogiri_1.12.3.bb,rubygems-nokogiri_1.11.5.bb